CONTRIBUTING.md

```diff
@@ -160,6 +160,15 @@ update /etc/hosts
 127.0.0.1 mesherylocal.layer5.io
 ```
 
+### UI Lint Rules 
+
+We are using ES-Lint to maintain code quality & consistency in our UI Code. To make sure your PR passes all the UI & ES-Lint Tests, please see below :
+
+- Remember to run `make run-ui-lint` & `make run-provider-ui-lint` if you are making changes in Meshery-UI & Provider-UI respectively.
+- The above commands will only fix some basic indenting rules for you, you will have to manually check your code to ensure there is no duplications, un-used variables or un-declared constants.
+- We will soon be adding Pre-Commit Hooks to make sure you get to know your errors before you commit the code.
```

Comment:
Given this PR is timed with PR #1107, perhaps, we should update this phrasing to say that we _do_ include a pre-commit hook. What do you think?

Reply:
That PR is incomplete, I am thinking to complete it once we have `golang-ci-lint` as golang-ci provides an option to run on local too.

Reply:
Ah, got it. 

Sounds like a good plan (`golang-ci-lint`)